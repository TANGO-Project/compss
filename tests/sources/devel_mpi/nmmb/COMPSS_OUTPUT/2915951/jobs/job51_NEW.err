[BINARY EXECUTION WRAPPER] ------------------------------------
[BINARY EXECUTION WRAPPER] CMD ERROR:
[BINARY EXECUTION WRAPPER] ------------------------------------
