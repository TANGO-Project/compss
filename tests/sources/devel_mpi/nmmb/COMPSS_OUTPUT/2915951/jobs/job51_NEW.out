[JAVA EXECUTOR] executeTask - Begin task execution
WORKER - Parameters of execution:
  * Method type: BINARY
  * Method definition: [BINARY=/gpfs/projects/bsc19/bsc19533/NMMB-BSC/nmmb-bsc-ctm-v2.0/PREPROC/VARIABLE/vegfrac.x]
  * Parameter types: java.lang.String java.lang.String java.lang.String java.lang.String
  * Parameter values: /scratch/tmp/b663fb84-ce0f-4725-8974-1ed5db6a4d72/s10r1b81-ib0/sandBox/job_51/llspl.000 /scratch/tmp/b663fb84-ce0f-4725-8974-1ed5db6a4d72/s10r1b81-ib0/sandBox/job_51/seamask /scratch/tmp/b663fb84-ce0f-4725-8974-1ed5db6a4d72/s10r1b81-ib0/sandBox/job_51/vegfrac /gpfs/projects/bsc19/bsc19533/NMMB-BSC/nmmb-bsc-ctm-v2.0/PREPROC/FIXED/../geodata/vegfrac/vegfracmnth
  * Parameter streams: UNSPECIFIED UNSPECIFIED UNSPECIFIED UNSPECIFIED
  * Parameter prefixes: null null null null

[BINARY INVOKER] Begin binary call to /gpfs/projects/bsc19/bsc19533/NMMB-BSC/nmmb-bsc-ctm-v2.0/PREPROC/VARIABLE/vegfrac.x
[BINARY INVOKER] On WorkingDir : /scratch/tmp/b663fb84-ce0f-4725-8974-1ed5db6a4d72/s10r1b81-ib0/sandBox/job_51
[BINARY INVOKER] BINARY CMD: /gpfs/projects/bsc19/bsc19533/NMMB-BSC/nmmb-bsc-ctm-v2.0/PREPROC/VARIABLE/vegfrac.x /scratch/tmp/b663fb84-ce0f-4725-8974-1ed5db6a4d72/s10r1b81-ib0/sandBox/job_51/llspl.000 /scratch/tmp/b663fb84-ce0f-4725-8974-1ed5db6a4d72/s10r1b81-ib0/sandBox/job_51/seamask /scratch/tmp/b663fb84-ce0f-4725-8974-1ed5db6a4d72/s10r1b81-ib0/sandBox/job_51/vegfrac /gpfs/projects/bsc19/bsc19533/NMMB-BSC/nmmb-bsc-ctm-v2.0/PREPROC/FIXED/../geodata/vegfrac/vegfracmnth 
[BINARY INVOKER] Binary STDIN: null
[BINARY INVOKER] Binary STDOUT: null
[BINARY INVOKER] Binary STDERR: null
[BINARY EXECUTION WRAPPER] ------------------------------------
[BINARY EXECUTION WRAPPER] Executing binary command
[BINARY EXECUTION WRAPPER] ------------------------------------
[BINARY EXECUTION WRAPPER] CMD EXIT VALUE: 0
[BINARY EXECUTION WRAPPER] ------------------------------------
[BINARY EXECUTION WRAPPER] CMD OUTPUT:
 Cycle data read from 
 /scratch/tmp/b663fb84-ce0f-4725-8974-1ed5db6a4d72/s10r1b81-ib0/sandBox/job_51/l
 lspl.000                                                                       
                                                                                
                    
 Cycle           24           4        2013           0  UTC
 vegfrac read data for month           4
 vegfracprog read data for month           5
0.000.000.000.000.000.000.00
0.000.000.000.000.000.000.00
0.000.000.000.000.000.010.01
0.000.250.000.000.260.150.00
0.000.350.240.000.240.140.00
0.000.810.000.000.270.010.00
0.000.000.000.000.010.380.00
0.000.000.000.010.010.540.00
0.000.000.210.410.040.000.00
0.000.000.710.000.000.710.00
0.000.000.810.000.000.000.00
0.000.000.020.000.570.000.00
0.000.000.130.000.000.000.00
0.000.000.050.000.000.000.00
0.000.000.000.000.000.000.00
0.000.000.000.000.000.000.00
0.000.000.010.000.010.010.01
0.000.010.000.010.010.010.01
0.010.010.010.010.010.010.01
 vegfrac written to ../output/vegfrac
 Enjoy your vegetation fraction!
[BINARY EXECUTION WRAPPER] ------------------------------------
[JAVA EXECUTOR] executeTask - End task execution
